<launch>
  <node name="iotbot_node" pkg="iotbot" type="iotbot_node" output="screen">
    <param name="track"               value="0.3"/>
    <param name="wheelBase"           value="0.3"/>
    <param name="wheelDiameter"       value="0.1"/>
    <param name="chFrontLeft"         value="3"/>
    <param name="chFrontRight"        value="1"/>
    <param name="chRearLeft"          value="2"/>
    <param name="chRearRight"         value="0"/>
    <param name="direction"           value="-1"/>
    <param name="enableYMotion"       value="1"/>
    <param name="gearRatio"           value="70"/>
    <param name="encoderRatio"        value="64"/>
    <param name="rpmMax"              value="140"/>
    <param name="kp"                  value="0.5"/>
    <param name="ki"                  value="5.0"/>
    <param name="kd"                  value="0.0"/>
    <param name="controlFrequency"    value="16000"/>
    <param name="lowPassInputFilter"  value="0.2"/>
    <param name="lowPassEncoderTicks" value="0.3"/>
  </node>
</launch>
